### PR TITLE
update @tableland/local dev-dep

### PR DESCRIPTION
**Pull request by joewagner**
This updates the tests for this package to use the most recent `@tableland/local` package, which will be necessary to prevent breakage when the go-tableland build without the JSON RPC API is released.

### CODE REVIEW

**Comment by joewagner** [comment]
@asutula Carson is OOO, if you have a few minutes today or tomorrow it would be nice to get this merged before EOD Friday so it can go into the next Ship.

**Comment by joewagner** [comment]
@asutula Looks like your approval doesn't work for this repo.  Maybe because you're not in CODEOWNERS?

**Comment by asutula** [comment]
Yea, looks like the repo settings require approval from a codeowner. Makes sense I guess, but we can always revisit that rule and decide if we want it still.

**Comment by joewagner** [comment]
> Yea, looks like the repo settings require approval from a codeowner. Makes sense I guess, but we can always revisit that rule and decide if we want it still.

The rule seems reasonable, but maybe we should have more than 2 codeowners in case one of them is on vacation?
Would you want to be a codeowner on this repo?

**Comment by carsonfarmer** [comment]
Yes, I think actually it should be a minimum of 3 for all CODEOWNER files.

**Review by carsonfarmer — APPROVED** [review]
Nice and simple, I like it!

**Pull request merged**
